examples/analytics/emr-on-eks/README.md

```diff
@@ -128,6 +128,22 @@ cd examples/analytics/emr-on-eks/examples/
 ./delete_emr_virtual_cluster_for_eks.sh
 ```
 
+To clean up your environment, destroy the Terraform modules in reverse order.
```

Comment:
I can't comment since the lines not changed but line 122 should change to `## Step 6: Cleanup`